tensilelite/Tensile/Components/StreamK.py

```diff
@@ -1910,8 +1916,22 @@ def graWorkGroup(self, writer, kernel, tPA, tPB):
         module.add(skUpdateDone)
         module.add(SMovB32(dst=sgpr("StreamKIter"), src=sgpr(sTmp+1), comment="Store current iteration"))
 
+        # Map SK index to WG
         module.add(self.skIndexToWG(writer, kernel, sTmp))
 
+        # Short circuit if alpha==0 (skip main loop and reading A/B, only do beta * C)
+        # To skip main loop in stream-k, we check if this WG is responsible for writing results (ie: WG starts tile)
+        # If WG starts tile then set LocalEnd=ItersPerTile to skip fixup step, and set loopCounter to 0 to skip main loop
+        # If WG does not start tile, skip to end of persistent loop to check for other SK tile
+        alphaLabel = Label("SKAlphaCheck", "")
+        module.add(SBranchIfNotZero("Alpha", kernel["ProblemType"]["ComputeDataType"], alphaLabel))
+        # Skip to end if not doing the global write
```

Comment:
What does this mean?
We need global write even if alpha==0, right?

Reply:
We need global write of beta * C (unless beta is 0, then it skips that part too)

Reply:
OK. I understand the logic.
StreamKLocalStart=0 WG does global write.
All other WGs with StreamKLocalStart>1, no global write, right?

Reply:
Yep that's right